src/Uno.UI.Runtime.Skia.Win32/Win32Host.cs

```diff
@@ -129,6 +132,42 @@ protected override void Initialize()
 		CompositionTargetTimer.Start();
 	}
 
+	private void SetProcessDpiAwareness()
+	{
+		// This call ensures that the application is properly marked as per-monitor DPI aware (PerMonitorV2).
+		// When launched via `dotnet.exe` (e.g., `dotnet MyApp.dll` or `dotnet run`), the host process is the .NET CLI runtime,
+		// which does NOT contain a DPI-awareness declaration in its embedded application manifest (RT_MANIFEST).
+		// Consequently, Windows assumes the process is DPI-unaware, applies bitmap scaling, and reports a logical DPI of 96,
+		// leading to incorrect pointer coordinates, layout glitches, and drag-and-drop offsets.
+		//
+		// In contrast, when launching a native application executable (e.g., `MyApp.exe`), the process inherits the
+		// DPI-awareness setting defined in the manifest block:
+		//
+		// <assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0" xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
+		//   <asmv3:application>
+		//     <asmv3:windowsSettings>
+		//       <dpiAwareness xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">PerMonitorV2</dpiAwareness>
+		//     </asmv3:windowsSettings>
+		//   </asmv3:application>
+		// </assembly>
+		//
+		// Important: Calling SetThreadDpiAwarenessContext() affects only the DPI behavior of user32/GDI calls **on that thread**,
+		// but it does NOT update the global DPI-awareness context of the process, nor does it affect how the OS scales
+		// the window or reports DPI-related metrics to frameworks like Skia or WinUI. To enable proper DPI scaling and
+		// accurate coordinate transformation, the process DPI-awareness must be set at launch or via SetProcessDpiAwarenessContext().
```

Comment:
That's not accurate. DPI scaling was already working accurately (i.e. actual scaling not just bitmap stretching), at least when running the executable directly. Also, Skia doesn't read DPI-related metrics, so I'm not sure what that's about.